README.md

```diff
@@ -40,9 +40,12 @@ $ pod install
 
 ### [Swift Package Manager](https://github.com/apple/swift-package-manager)
 
-To add the AEPEdgeIdentity Package to your application, from the Xcode menu select:
+To add the AEPEdgeIdentity Package to your application, from the Xcode 13.x menu select:
```

Comment:
nit: I'd suggest not including the version number here as it can lead to confusion once later versions of Xcode, such as 14, are released.